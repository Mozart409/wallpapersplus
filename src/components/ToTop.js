import React from "react";
import ScrollToTop from "react-scroll-up";

function ToTop() {
  return (
    <ScrollToTop showUnder={160}>
      <span
      className="bg-blue-700 text-white p-4 m-2 round hover:shadow-xl"
      >UP</span>
    </ScrollToTop>
  );
}
export default ToTop;
