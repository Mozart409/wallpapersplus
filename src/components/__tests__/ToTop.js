import React from "react";
import renderer from "react-test-renderer";

import ToTop from "../ToTop";

describe("ToTop", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ToTop />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
