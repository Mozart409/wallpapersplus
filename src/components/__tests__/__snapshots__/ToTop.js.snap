// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToTop renders correctly 1`] = `
<div
  className="base-back-to-up fade-hide"
  onClick={[Function]}
>
  <div
    className="back-to-up-default"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
  <div
    className="to-up-percent"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    0%
  </div>
</div>
`;
